src/huggingface_hub/hf_api.py

```diff
@@ -1858,6 +1858,8 @@ def create_commit(
                 If `create_pr` is `True`, returns the URL to the newly created Pull Request
                 on the Hub. Otherwise returns `None`.
         """
+        if commit_message is None or len(commit_message) == 0:
```

Comment:
```suggestion
        if commit_message:
```

Reply:
Have been burned too many times by relying on Python bool-conversion magic so I prefer explicit tests :-) But feel free to merge the suggestion if it fits the style of huggingface_hub better.